src/interface/mod.rs

```diff
@@ -183,7 +183,10 @@ impl Interface {
     }
 
     /// Get a [`MappingRef`] reference of the path if the endpoint is present in the interface.
-    pub(crate) fn as_mapping_ref(&self, path: &MappingPath) -> Option<MappingRef<&Interface>> {
+    pub(crate) fn as_mapping_ref<'a>(
+        &'a self,
+        path: &'a MappingPath,
+    ) -> Option<MappingRef<&Interface>> {
```

Comment:
Maybe I'm missing something, but shouldn't we specify also a lifetime in the return value?

Reply:
No